builder/dispatchers.go

```diff
@@ -52,7 +60,8 @@ func env(b *Builder, args []string, attributes map[string]bool, original string)
 	for j := 0; j < len(args); j++ {
 		// name  ==> args[j]
 		// value ==> args[j+1]
-		newVar := args[j] + "=" + args[j+1] + ""
+		newValue := b.replaceEnv(args[j+1])
+		newVar := args[j] + "=" + newValue + ""
```

Comment:
Just confused as to what is happening, but I think I understand now. Is the string append at the end necessary?